src/component/vue-hcaptcha.vue

```diff
@@ -4,9 +4,12 @@
   </div>
 </template>
 <script>
-const CaptchaScript = (cb) => {
+const CaptchaScript = (cb, lang) => {
+    if(typeof lang === 'undefined') { // If not set, set english as default
```

Comment:
I don't think this is the behavior you want: setting hl will force a language override, turning off browser autodetection.

Thus, this logic will force all users into English if you don't specify hl.

The correct behavior would be to simply not add hl if not specified.

Reply:
@e271828- is correct, hl should only be added to the string if the language was specified.